.github/workflows/translations.yml

```diff
@@ -0,0 +1,28 @@
+name: Translations Check
+on:
+  pull_request:
+    paths:
+      - "translations/**"
+env:
+  GOPROXY: https://proxy.golang.org
+  GO_VERSION: 1.16.4
```

Comment:
update "goland updater" to include this yaml file too

https://github.com/medyagh/minikube/blob/b82c56cbcf03910087a1a153faa327d9326ecb73/hack/update/golang_version/update_golang_version.go#L73